front-end/README.md

```diff
@@ -0,0 +1,27 @@
+# FrontEnd
```

Comment:
Looks good, this can be added to the README at the top of the project, it's nicer to keep the documentation close together. In my opinion